src/components/layout/nav/PrivateMenu.tsx

```diff
@@ -61,25 +62,32 @@ export default function PrivateMenu() {
   }
 
   const title = `${session?.user?.name}\n(${session?.user?.email})`
-  const lettersAvatar = `${session.user?.given_name?.charAt(0) || session.user?.email?.charAt(0)}${
-    session.user?.family_name?.charAt(0) || session.user?.email?.charAt(1)
-  }`.toUpperCase()
+
+  const getAvatar = () => {
+    const { data: user } = getCurrentPerson(!!router.query?.register)
```

Comment:
I think the `Boolean` cast is way more clearer than `!!` 
```suggestion
    const { data: user } = getCurrentPerson(Boolean(router.query?.register))
```